Logging/composer.json

```diff
@@ -12,23 +12,21 @@
         "yoast/phpunit-polyfills": "^1.0",
         "phpspec/prophecy": "^1.10.3",
         "squizlabs/php_codesniffer": "2.*",
-        "phpdocumentor/reflection": "^3.0||^4.0",
+        "phpdocumentor/reflection": "^3.0||^4.0||^5.3",
         "erusev/parsedown": "^1.6",
+        "fig/log-test": "^1.0",
         "google/cloud-storage": "^1.3",
         "google/cloud-bigquery": "^1.0",
         "google/cloud-pubsub": "^1.0",
         "opis/closure": "^3"
     },
     "provide": {
-        "psr/log-implementation": "1.0|2.0"
+        "psr/log-implementation": "^1.0|^2.0|^3.0"
     },
     "suggest": {
         "ext-grpc": "The gRPC extension enables use of the performant gRPC transport",
         "ext-protobuf": "Provides a significant increase in throughput over the pure PHP protobuf implementation. See https://cloud.google.com/php/grpc for installation instructions.",
-        "psr/log": "For using the PSR logger. Currently supports versions 1 and 2."
-    },
-    "conflict": {
-        "psr/log": ">=3"
```

Comment:
This is still the case, until we update `PsrLogger` to be compatible with the interface from `psr/log: 3`, which we can't do until we drop support for PHP 7.0 and below. So this should be added back.

Reply:
This one was just about the duplicate `"conflict"` section 😅